test/models/work_test.rb

```diff
@@ -0,0 +1,91 @@
+require "test_helper"
+
+describe Work do
+  let(:work) { works(:work_one) }
+
+  describe 'validations' do
+    it "must be valid with all valid inputs" do
+      work.valid?.must_equal true
+    end
+
+    it "does not accept a nil category input" do
+      work.category = nil
+      work.valid?.must_equal false
+    end
+
+    it "must not accept an empty string category input" do
+      work.category = ""
+      work.valid?.must_equal false
+    end
+
+    it "must require unique title by category" do
+      work
+      new_work = Work.create(category: "book", title: "book title 1")
+      new_work.valid?.must_equal false
+
+      album = works(:work_eleven)
+      new_album = Work.create(category: "album", title: "album title 1")
+      new_album.valid?.must_equal false
+
+      movie = works(:work_sixteen)
+      new_movie = Work.create(category: "movie", title: "movie title 1")
+      new_movie.valid?.must_equal false
+    end
+
+    it "does not accept a nil title input" do
+      work.title = nil
+      work.valid?.must_equal false
+    end
+
+    it "must not accept an empty string title input" do
+      work.title = ""
+      work.valid?.must_equal false
+    end
+
+  end
+
+  describe 'relations' do
+    it "must have a relationship between work and votes" do
+      work.votes.count.must_equal 4
+    end
+
+    it "must respond to votes" do
+      work.must_respond_to :votes
+    end
+  end
+
+
+  describe 'vote_count' do
+    it "counts the votes" do
+      work.votes.count.must_be_kind_of Integer
+
+      work.vote_count.must_equal 4
+    end
+
+    it "returns 0 for works with no votes" do
+      works(:work_three).vote_count.must_equal 0
+    end
+  end
+
+  describe 'ordered_works' do
```

Comment:
You also need to test when:
1.  There are no works
1.  There are no votes
1.  One category has no votes.